Intersect.Client/Entities/Entity.cs

```diff
@@ -1323,6 +1317,41 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
             );
         }
 
+        private bool DrawNameRequirements()
```

Comment:
Almost all of this logic should already be in ShouldDrawName or ShouldNotDrawName if it exists. I don't see how this method is useful, it looks like it is rechecking things unnecessarily.

Reply:
there wasn't nor there is a `ShouldNotDrawName` method for the logic behind this so far

moved over the logic to ShouldDrawName